### PR TITLE
Update docs to use pretty link for installer

### PR DESCRIPTION
**Pull request by lakshmi-kannan**

### CODE REVIEW

File: docs/source/install/deb.rst [hunk 1]
```diff
@@ -2,9 +2,15 @@ Ubuntu / Debian
...
-instructions below.
+the :doc:`Reference deployment </install/overview>`.
+
+.. rubric:: TL;DR
+
+That's OK! You're busy, we get it. How do you just get started? Get yourself a clean box, and run this command:
+
+::
+
+   curl -sSL https://stackstorm.com/packages/install.sh | bash -s -- --user=st2admin --password=<CHANGME>
```
Comment (emedvedev) [inline docs/source/install/deb.rst]: > CHANGME
A tiny typo here. :)

**Comment by dzimine** [comment]
nit pick is "clean box"; we don't require a clean. But I wouldn't bother PR for this. 

+1

File: docs/source/install/deb.rst [hunk 2]
```diff
@@ -2,9 +2,15 @@ Ubuntu / Debian
 ===============
 
 This guide provides step-by step instructions for installing StackStorm on a single Ubuntu/Debian system per
-the :doc:`Reference deployment </install/overview>`. The script `st2bootstrap-deb.sh
-<https://github.com/StackStorm/st2-packages/blob/master/scripts/st2bootstrap-deb.sh>`_ codifies the
-instructions below.
+the :doc:`Reference deployment </install/overview>`.
+
+.. rubric:: TL;DR
```
Comment (Kami) [inline docs/source/install/deb.rst]: We should start utilizing `include` directive more often for things like that (check some of my previous PRs of how I do it).

Especially the installer needs some include love, way too much duplicated stuff :P

Comment (Kami) [inline docs/source/install/deb.rst]: Besides that, :+1:

**Pull request merged**